packages/shared/assetdb.ts

```diff
@@ -12,6 +12,7 @@ function getAssetDir(userId: string, assetId: string) {
 
 export const zAssetMetadataSchema = z.object({
   contentType: z.string(),
+  fileName: z.string(),
```

Comment:
This needs to be nullable for backward compatibility. Otherwise hoarder throws errors when we attempt to read the metadata for assets uploaded before this commit.

```suggestion
  fileName: z.string().nullish(),
```